torch/hub.py

```diff
@@ -423,7 +423,7 @@ def _download_url_to_file(url, dst, hash_prefix=None, progress=True):
             _download_url_to_file will be removed in after 1.3 release')
     download_url_to_file(url, dst, hash_prefix, progress)
 
-def load_state_dict_from_url(url, model_dir=None, map_location=None, progress=True, check_hash=False):
+def load_state_dict_from_url(url, model_dir=None, file_name=None, map_location=None, progress=True, check_hash=False):
```

Comment:
adding an arg in the middle is BC breaking

Reply:
I found it more sensible to add after `model_dir`, i am okay with changing the position if it will maintain compatibility though